WPFSKillTree/SkillTreeFiles/Compute.cs

```diff
@@ -2310,7 +2310,7 @@ public static void Initialize(SkillTree skillTree, ItemAttributes itemAttrs)
 
             CoreAttributes();
 
-            Implicit = new AttributeSet(skillTree.ImplicitAttributes(Global));
+            Implicit = new AttributeSet(SkillTree.ImplicitAttributes(Global, skillTree.Level));
```

Comment:
Ah, it's static. Didn't notice case change from skillTree to SkillTree in call to ImplicitAttributes.
Ok. It doesn't matter. SillTree.ImplicitAttribtues will be gone soon or later.
But for sake of correctness, you should pass Level (Compute.Level) to SkillTree.ImplicitAttributes() and not SkillTree.Level, or do level range validation on SkillTree.Level, otherwise calculations will be incorrect for level values outside of 1 ... 100 range.
Just change 2 lines in Compute.Initialize to:
...
Level = SkillTree.Level;
...
Implicit = new AttributeSet(SkillTree.ImplicitAttributes(Global, Level));